docs/en/setup/backend/opentelemetry-receiver.md

```diff
@@ -18,7 +18,7 @@ To activate the `otlp` handler and relevant rules of `istio`:
 receiver-otel:
   selector: ${SW_OTEL_RECEIVER:default}
   default:
-    enabledHandlers: ${SW_OTEL_RECEIVER_ENABLED_HANDLERS:"otlp"}
+    enabledHandlers: ${SW_OTEL_RECEIVER_ENABLED_HANDLERS:"otlp-metrics"}
```

Comment:
@kezhenxu94 This was missed when adding new handler, right?

Reply:
> @kezhenxu94 This was missed when adding new handler, right?

Yes thanks!